modules/log.cpp

```diff
@@ -268,15 +269,35 @@ CString CLogMod::GetServer()
 
 bool CLogMod::OnLoad(const CString& sArgs, CString& sMessage)
 {
+	CString sMyArgs = sArgs;
```

Comment:
Well, do you need the copy?
Awaystore parses differently, it uses Token(0) every time, and chops the first token out of string every time. So it needs the string to be modifiable.